Automattic/woocommerce-payments
# Add Horizontal list of dispute details to the transaction page

Fixes #6924
Fixes #7164

#### Changes proposed in this Pull Request

![Screenshot 2023-09-01 at 3 26 28 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/514631a2-f501-433c-abd6-501d484b4dbe)

A row layout will be added to the DisputeDetails Component showing the key attributes of the dispute. This will show at all times regardless of the status of the dispute.

 - `Dispute Amount` ~in the store currency. If the store accepts multicurrency payments and the customer paid $70.29 AUD. The amount would be shown as $45.45 USD~
 - `Disputed On` The date the dispute was opened Reason The reason for the dispute. This includes a ClickTooltip component showing more information about the reason on click.
 - `Respond By` The evidence required by date.
   - Default: Shows date/time with number of days left Aug 18, 2023 11:59 PM (12 days left to respond)
   - Less than 7 days: As above but text showing days left is yellow (6 days left to respond) WP Yellow/Yellow 30 ( #DBA617 )
   - Less than 3 days: As above but text showing days left is red (2 days left to respond) WP Gutenberg/Alert Red ( ##CC1818 )
   - Less than 1 day: Same as Less than 3 days but text now reads (Last day today)
   - 
No details are shown once evidence is submitted or the response date has passed.
 
![Screenshot 2023-09-01 at 3 26 38 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/77940eac-5b2d-41e9-b2c3-a5ffd986f55d)


#### Testing instructions

1. Enable feature flag by running `update_option( '_wcpay_feature_dispute_on_transaction_page', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/) or by modifying your database table `wp_options` directly.
2. Load a Transaction Details page of an active dispute (`/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails&id=pi_abc&transaction_id=txn_abc&transaction_type=charge`).
3. See the Dispute Details. 
4. Test with differing currencies, dispute reasons and respond by dates.

-------------------


**Post merge**
